researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/task/tracked/TrackedDataSelectedItemsInterface.java

```diff
@@ -0,0 +1,17 @@
+package org.sagebionetworks.bridge.researchstack.task.tracked;
+
+import org.researchstack.backbone.result.StepResult;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 3/23/17.
+ */
+
+public interface TrackedDataSelectedItemsInterface {
```

Comment:
Those are good thoughts, and I think you are on to something. I'm not 100% this class is even needed.  It was created as I converted from iOS, and need to match parity with it's use of multiple inheritance. I'm going to put this on my radar, and avoid refactoring it just yet,  and as I finish the TrackedDataTask and it's unit tests, I will re-evaluate what this should be called and how it should operate.